themes/README.md

```diff
@@ -26,7 +26,7 @@
   <a href="https://npmcharts.com/compare/gatsby?minimal=true">
     <img src="https://img.shields.io/npm/dm/gatsby.svg" alt="Downloads per month on npm." />
   </a>
-  <a href="https://www.gatsbyjs.org/contributing/how-to-contribute/">
+  <a href="https://www.gatsbyjs.org/contributing/how-to-open-a-pull-request/">
```

Comment:
the `how to contribute` is the same as in:

https://github.com/gatsbyjs/gatsby/blob/master/README.md